langserve/server.py

```diff
@@ -333,6 +335,52 @@ def _get_base_run_id_as_str(
         raise AssertionError("No run_id found for the given run")
 
 
+def _json_encode_response(obj: BaseModel, *, is_batch: bool = False) -> JSONResponse:
```

Comment:
micronit that is not necessary: for readability, it could be nice to give some subclass to all our requests/responses that can then be referenced here

Reply:
see if you like current version